common/lib/common-utils/package.json

```diff
@@ -95,6 +94,7 @@
 		"@rushstack/eslint-config": "^2.5.1",
 		"@types/base64-js": "^1.3.0",
 		"@types/benchmark": "^2.1.0",
+		"@types/events": "^3.0.0",
```

Comment:
This being a dep instead of a dev dep may be intentional. @curtisman FYI.

Reply:
Oh I see #3449 now, but I don't understand why it would be required?

Reply:
Reverting, this isn't the focus of this PR so we can consider it later.